corehq/apps/repeaters/models.py

```diff
@@ -324,6 +261,13 @@ class FormRepeater(Repeater):
 
     """
 
+    class Formats(object):
+        formats = {
+            'form_xml': (FormRepeaterXMLPayloadGenerator, _('XML')),
+            'form_json': (FormRepeaterJsonPayloadGenerator, _('JSON')),
+        }
+        default_format = 'form_xml'
```

Comment:
can you do this instead?

```py
class Repeater(Document):
    _formats = [Formats]

    @property
    def formats(self):
        return _formats

    @formats.setter(self, formats)
        self._formats = formats
```
I guess that doesn't work with class instantiation.